src/hooks/useChanochaBigBrainCells.ts

```diff
@@ -0,0 +1,3 @@
+export function useChanochaBigBrainCells(): number {
+  return 84000 * 1000000
```

Comment:
ลุงเขาบอกว่า 840,000 ล้านอ่ะครับ ไม่ฝากแก้ครับ เดี๋ยวผมแก้เอง 555

```suggestion
  return 840000 * 1000000
```

Reply:
แง้ ขอบคุณครับ 555